x/auth/client/rest/rest_test.go

```diff
@@ -480,10 +480,16 @@ func (s *IntegrationTestSuite) testQueryIBCTx(txRes sdk.TxResponse, cmd *cobra.C
 func (s *IntegrationTestSuite) TestLegacyRestErrMessages() {
 	val := s.network.Validators[0]
 
+	// Write client state json to temp file, used for an IBC message.
+	clientStateJSON := testutil.WriteToNewTempFile(
```

Comment:
cc @amaurymartiny 

changing this since the cli command is being moved to a more generic one